compose/config/config_schema_v3.0.json

```diff
@@ -205,12 +205,13 @@
         "interval": {"type":"string"},
         "timeout": {"type":"string"},
         "retries": {"type": "number"},
-        "command": {
```

Comment:
Why is "test" preferrable to "command"? The references I can find for engine use "command" or "cmd", not "test".